src/Main.jsx

```diff
@@ -62,9 +78,11 @@ export default class Main extends Component {
       ? `${slugField.attributes.api_key}.${locale}`
       : slugField.attributes.api_key;
 
+    if (fieldPath) urlParts.push(plugin.getFieldValue(fieldPath));
+
     this.setState({
       slugField,
-      initalValue: plugin.getFieldValue(fieldPath),
+      initalValue: urlParts.join('/'),
```

Comment:
Typo. Should be `initialValue`. I know it's not yours but let's fix it either way :)